src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java

```diff
@@ -86,7 +86,7 @@ public boolean canAccessCaseStateWithCriteria(final String caseState,
         boolean hasAccess = hasAccessControlList(accessProfiles, criteria, stateACLs);
 
         if (!hasAccess) {
-            LOG.error(NO_ROLE_FOUND, "caseState",
+            LOG.debug(NO_ROLE_FOUND, "caseState",
```

Comment:
In last 7 days this is logged ~6m times, it's part of the standard filtering so we shouldn't need to log everything.